logging/measure/model/apitrace/apitrace.proto

```diff
@@ -0,0 +1,19 @@
+syntax = "proto3";
+
+// Package apitrace provides data model for binary API trace informations.
+package apitrace;
+
+message Trace {
+    message Entry {
+        uint64 index = 1;                       // Trace index
+        string msg_name = 2;                    // Message name (or any other string)
+        uint64 duration = 3;                    // Duration as integer
```

Comment:
You probably meant `... as nanoseconds`, the integer is obvious from definition.